commit 84db3de5a39452d97616ef3dcc5940929bf7d639
Author: Gavin Shan <gwshan@linux.vnet.ibm.com>
Date:   Fri Mar 17 00:48:25 2017 +0000

    PCI: Do any VF BAR updates before enabling the BARs
    
    [ Upstream commit f40ec3c748c6912f6266c56a7f7992de61b255ed ]
    
    Previously we enabled VFs and enable their memory space before calling
    pcibios_sriov_enable().  But pcibios_sriov_enable() may update the VF BARs:
    for example, on PPC PowerNV we may change them to manage the association of
    VFs to PEs.
    
    Because 64-bit BARs cannot be updated atomically, it's unsafe to update
    them while they're enabled.  The half-updated state may conflict with other
    devices in the system.
    
    Call pcibios_sriov_enable() before enabling the VFs so any BAR updates
    happen while the VF BARs are disabled.
    
    [bhelgaas: changelog]
    Tested-by: Carol Soto <clsoto@us.ibm.com>
    Signed-off-by: Gavin Shan <gwshan@linux.vnet.ibm.com>
    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
    
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1430_84db3de5a39452d97616ef3dcc5940929bf7d639.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1430_84db3de5a39452d97616ef3dcc5940929bf7d639.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
